### PR TITLE
added pancake recipe

### PR DESCRIPTION
**Pull request by byjennybrown**
review pancake recipe

### CODE REVIEW

File: pancake.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Ingredients
+1 1/2 cup flour
+2 teaspoon baking powder
+1 teaspoon baking soda
+1/2 salt
+1 cup of milk
+1 egg
+1 tablespoon olive oil
+
```
Comment (mtiberti) [inline pancake.txt:11]: can we have butter instead of olive oil?

**Review by mtiberti — CHANGES_REQUESTED** [review]
Thanks! I've asked to change an ingredient - see below

File: pancake.txt (hunk 1 above)
Comment (byjennybrown) [reply pancake.txt:11]: yes

Comment (mtiberti) [reply pancake.txt:11]: thanks!

**Review by mtiberti — APPROVED** [review]

**Pull request merged**